sdk/core/azure-core/inc/azure/core/internal/tracing/service_tracing.hpp

```diff
@@ -29,7 +29,7 @@ namespace Azure { namespace Core { namespace Tracing { namespace _internal {
   private:
     std::shared_ptr<Span> m_span;
 
-    friend class DiagnosticTracingFactory;
+    friend class ContextAndSpanFactory;
```

Comment:
There are some readme/docs that we'd want to update based on the renames in this PR. For example. Though that's not release blocking and can be done separately.
https://github.com/Azure/azure-sdk-for-cpp/blob/main/doc/DistributedTracing.md

Reply:
Already done.